mail_tracking/models/mail_tracking_email.py

```diff
@@ -166,15 +166,20 @@ def _find_allowed_tracking_ids(self):
         partner_ids = self.env["res.partner"]._search(
             [("id", "in", [x for x in partner_ids if x])]
         )
-        return [
-            x[0]
-            for x in msg_linked
-            if (x[1] in msg_ids)  # We can read the linked message
-            or (
-                not any({x[1], x[2]}) and x[3] in partner_ids
-            )  # No linked msg/mail but we can read the linked partner
-            or (not any({x[1], x[2], x[3]}))  # No linked record
-        ]
+        res = []
+
+        for tracking_id, message_id, partner_id in msg_linked:
+            # We can read the linked message
+            linked_message = message_id in msg_ids
+            # No linked msg/mail but we can read the linked partner
+            linked_partner = not any({tracking_id, message_id}) and partner_id in partner_ids
```

Comment:
Sorry, although the issue is real, this fix isn't right as it doesn't address the issue correctly. You won't ever enter this condition or the next one :wink: 

Please, follow: https://github.com/OCA/social/pull/1148